.github/workflows/build.yml

```diff
@@ -0,0 +1,75 @@
+name: Build
+
+on: [push, pull_request]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  win-build:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Environment
+      run: |
+        echo ${env:BOOST_ROOT_1_72_0}
+        echo "BOOST_ROOT=${env:BOOST_ROOT_1_72_0}" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        #choco install boost-msvc-14.2
+
+    - name: Build
+      run: |
+        cmake -B build
+        cmake --build build -j2 --config ${env:BUILD_TYPE}
```

Comment:
What is the logic for using '-j' option?

Reply:
Github Actions has VM with 2 CPUs and no other options are available for shared runners.

Reply:
I see. However - why don't just let it decide automatically? BTW, macOS runner appears to have 3 CPUs :)

Reply:
3 CPUs seem to be something new.

Reply:
```suggestion
        cmake --build build --config ${env:BUILD_TYPE}
```